ITEM_en.tsv

```diff
@@ -4053,55 +4053,55 @@ ITEM_20150401_004415	Superior Infantry Gauntlets
 ITEM_20150401_004416	Superior Infantry Greaves
 ITEM_20150401_004417	Superior Infantry Leggings
 ITEM_20150401_004418	Superior Infantry Armor
-ITEM_20150401_004419	Librarian Gloves
-ITEM_20150401_004420	Librarian Boots
-ITEM_20150401_004421	Librarian Pants
-ITEM_20150401_004422	Librarian Robe
-ITEM_20150401_004423	Veteran Gloves
-ITEM_20150401_004424	Veteran Boots
-ITEM_20150401_004425	Veteran Pants
-ITEM_20150401_004426	Veteran Tunic
-ITEM_20150401_004427	Knight Gauntlets
-ITEM_20150401_004428	Knight Tunic
-ITEM_20150401_004429	Knight Leggings
-ITEM_20150401_004430	Knight Armor
+ITEM_20150401_004419	Librarian's Gloves
+ITEM_20150401_004420	Librarian's Boots
+ITEM_20150401_004421	Librarian's Pants
+ITEM_20150401_004422	Librarian's Robe
+ITEM_20150401_004423	Veteran's Gloves
+ITEM_20150401_004424	Veteran's Boots
+ITEM_20150401_004425	Veteran's Pants
+ITEM_20150401_004426	Veteran's Tunic
+ITEM_20150401_004427	Knight's Gauntlets
+ITEM_20150401_004428	Knight's Tunic
+ITEM_20150401_004429	Knight's Leggings
+ITEM_20150401_004430	Knight's Armor
 ITEM_20150401_004431	Superior Grima Gloves
 ITEM_20150401_004432	Superior Grima Boots
 ITEM_20150401_004433	Superior Grima Pants
 ITEM_20150401_004434	Superior Grima Robe
-ITEM_20150401_004435	Magnus Gloves
-ITEM_20150401_004436	Magnus Boots
-ITEM_20150401_004437	Magnus Robe
+ITEM_20150401_004435	Magnus' Gloves
+ITEM_20150401_004436	Magnus' Boots
+ITEM_20150401_004437	Magnus' Robe
 ITEM_20150401_004438	Superior Boots
-ITEM_20150401_004439	Ronesa Wooden Necklace
-ITEM_20150401_004440	Ronesa Iron Necklace
-ITEM_20150401_004441	Ronesa Steel Necklace
-ITEM_20150401_004442	Ronesa Talisman
-ITEM_20150401_004443	Ronesa Klaipeda Chain
+ITEM_20150401_004439	Ronesa's Wooden Necklace
+ITEM_20150401_004440	Ronesa's Iron Necklace
+ITEM_20150401_004441	Ronesa's Steel Necklace
+ITEM_20150401_004442	Ronesa's Talisman
+ITEM_20150401_004443	Ronesa Klaipeda's Chain
 ITEM_20150401_004444	Ritual Necklace
 ITEM_20150401_004445	Superior Ritual Necklace
 ITEM_20150401_004446	Key Pendant
-ITEM_20150401_004447	Superior key Pendant
-ITEM_20150401_004448	Sage Necklace
-ITEM_20150401_004449	Ligh Ronesa Bangle
-ITEM_20150401_004450	Superior Ronesa Bangle
-ITEM_20150401_004451	Ronesa Ring Bracelet
-ITEM_20150401_004452	Ronesa Noble Bracelet
-ITEM_20150401_004453	Superior Health Bangle
-ITEM_20150401_004454	Superior Magic Bangle
-ITEM_20150401_004455	Superior Accuracy Bangle
+ITEM_20150401_004447	Superior Key Pendant
+ITEM_20150401_004448	Sage's Necklace
+ITEM_20150401_004449	Light Ronesa's Bangle
+ITEM_20150401_004450	Superior Ronesa's Bangle
+ITEM_20150401_004451	Ronesa's Ring Bracelet
+ITEM_20150401_004452	Ronesa's Noble Bracelet
+ITEM_20150401_004453	Superior Bangle of Health
+ITEM_20150401_004454	Superior Magical Bangle
```

Comment:
Superior Magical Bangle -> Superior Bangle of Magi
I don't know what this item does, but the Korean string tells it is Magi.